definitions/divorce/json/CaseEvent/CaseEvent-share-a-case-nonprod.json

```diff
@@ -10,7 +10,6 @@
     "PostConditionState": "*",
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/petition-issue-fees",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/process-pba-payment",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/allow-share-a-case",
```

Comment:
![image](https://user-images.githubusercontent.com/5647337/106734490-5f0c8580-6613-11eb-9785-a9e189a0a076.png)

We don't know why it happens. Lets make sure we don't block others and investigate it from the branch.